.github/workflows/docker-build-and-push.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 permissions:
   id-token: write
-  contents: read
+  contents: write
```

Comment:
Just curious how this worked before without a `write` permission for contents?

Reply:
Perviously we only needed `read` to checkout the repo.  Adding the Docker SBOM push to GitHub mean that we now need to `write` to the repo's dependency graph.

By default, all our workflows have [permissive GitHub workflow permissions](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token) which grants us `write` to almost everything, but as soon as you add a `permission` block (which we need to for the `id-token` to support OIDC auth to AWS) you then have to specify everything.

Reply:
Thanks for the explanation!